rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/CovidRDTBarcodeFactory.java

```diff
@@ -40,10 +40,8 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 
                 String[] individualVals = splitCSV(barcodeVals);
                 populateRelevantFields(individualVals);
-                moveToNextStep(convertDate(individualVals[1], "YYYY-MM-dd"));
-            } catch (JSONException e) {
-                Timber.e(e);
-            } catch (ParseException e) {
+                moveToNextStep(data, convertDate(individualVals[1], "YYYY-MM-dd"));
```

Comment:
`sensorTriggered` value is already extracted https://github.com/onaio/rdt-standard/blob/9f2f4e819f416fa4b3e5d9b8e6dd0edfbab9ddde/rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/widget/CovidRDTBarcodeFactory.java#L65. You can pass that instead.

Reply:
how do i get individualVals[4] in `moveToNextStep` method